viadot/flows/mindful_to_adls.py

```diff
@@ -110,13 +110,13 @@ def mind_flow(self) -> Flow:
 
         add_timestamp.bind(file_names, sep=self.sep, flow=self)
 
-        adls_bulk_upload(
+        adls_bulk_upload.bind(
```

Comment:
Please assign it to something and add it to the section at the bottom - `set_upstream`.